superduperdb/ext/transformers/model.py

```diff
@@ -1,9 +1,14 @@
+# ruff: noqa: E402
 import dataclasses as dc
 import functools
 import re
 import typing as t
 import warnings
 
+from superduperdb.misc.annotations import requires_modules
+
+requires_modules('transformers>=4.29.1')
```

Comment:
Why does it need to be all in 1 `str`? It would make your life easier to do

`requires_package('transformers', '4.29.1', '5.1.0')`

Or something like that

Reply:
I mean the requirement is from pyproject and requirements.txt compatible 
also Hugging face also doing exactly the same.